rootconf/default/lib/onie/onie-uninstaller-common

```diff
@@ -80,8 +80,13 @@ erase_part()
 {
     local blk_dev="$1"
     local part="$2"
+    # check if we have an mmcblk device
+    local blk_suffix=
+    echo ${blk_dev} | grep -q mmcblk && blk_suffix="p"
+    # check if we have an nvme device
+    echo ${blk_dev} | grep -q nvme && blk_suffix="p"
 
-    erase_block_device "${blk_dev}$part"
+    erase_block_device "${blk_dev}$blk_suffix$part"
```

Comment:
Thanks so much for fixing this!  Looks like this is a long standing bug for mmcblk devices.